doc/source/tune/why_tune.rst

```diff
@@ -0,0 +1,30 @@
+Tune offers cutting-edge optimization algorithms.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As a user, you're probably looking into hyperparameter optimization because you want to quickly increase your model performance.
+
+Tune enables you to leverage a variety of these cutting edge optimization algorithms, reducing the cost of tuning by `aggressively terminating bad hyperparameter evaluations <tune-scheduler-hyperband>`_, intelligently :ref:`choosing better parameters to evaluate <tune-search-alg>`, or even :ref:`changing the hyperparameters during training <tune-scheduler-pbt>` to optimize hyperparameter *schedules*.
```

Comment:
nit: maybe make this a bullet list? this isn't a big deal, though

Reply:
The emphasis on *schedules* is a bit odd if we don't talk about it afterwards. I would just remove it. 
If we added bullet points, I would expect this in all sub chapters, so I it's good the way it is now.

Reply:
```suggestion
Tune enables you to leverage a variety of these cutting edge optimization algorithms, reducing the cost of tuning by `aggressively terminating bad hyperparameter evaluations <tune-scheduler-hyperband>`_, intelligently :ref:`choosing better parameters to evaluate <tune-search-alg>`, or even :ref:`changing the hyperparameters during training <tune-scheduler-pbt>` to optimize hyperparameter schedules.
```